sdk/eventgrid/azure-eventgrid/azure/eventgrid/_messaging_shared.py

```diff
@@ -13,18 +13,22 @@
 
 
 import json
-from azure.core.exceptions import raise_with_traceback
 
-def _get_json_content(obj):
+def _get_json_content(obj): # pylint: disable=inconsistent-return-statements
     """Event mixin to have methods that are common to different Event types
     like CloudEvent, EventGridEvent etc.
+
+    :param obj: The object to get the JSON content from.
+    :type obj: Any
+    :rtype: Any
+    :return: The JSON content from the object.
```

Comment:
would this fit here ?
```suggestion
    :return: The JSON content from the object.
    :raises ValueError: If the provided JSON is invalid.
```